nautobot/users/filters.py

```diff
@@ -29,17 +36,45 @@ class UserFilterSet(BaseFilterSet):
             "email": "icontains",
         },
     )
-    group_id = django_filters.ModelMultipleChoiceFilter(
+    groups_id = django_filters.ModelMultipleChoiceFilter(
         field_name="groups",
         queryset=Group.objects.all(),
         label="Group",
     )
-    group = django_filters.ModelMultipleChoiceFilter(
+    groups = django_filters.ModelMultipleChoiceFilter(
         field_name="groups__name",
         queryset=Group.objects.all(),
         to_field_name="name",
         label="Group (name)",
     )
+    has_changes = RelatedMembershipBooleanFilter(
+        field_name="changes",
+        label="Has Changes",
+    )
+    changes = NaturalKeyOrPKMultipleChoiceFilter(
+        to_field_name="user_name",
```

Comment:
This feels weird to me - so this would be `/api/users/user?changes=<some_user_name>`? That doesn't make a lot of sense to me. Maybe `changes` should just be a ModelMultipleChoiceFilter for now?

Reply:
In that case we would only have `changes_id` right?

Reply:
`changes` would still be the appropriate filter name, it would just be UUID-only for now.